Unable to connect to MySQL

Following the README, I clone the project and then perform the following steps:
1. `mvn install` - no issues here
2. `mvn jetty:run` - no issues; I'm able to access the web UI as expected. This is using Derby as indicated in the README.
3. Run local MySQL:
    - `docker run --name hapi-mysql -e MYSQL_ROOT_PASSWORD=mypassword -p 3306:3306 -d mysql:5`
    - Connect to local MySQL with DB client and run the following to create user and hapi_dstu3 database:
```
CREATE USER 'hapiDbUser'@'localhost' IDENTIFIED BY 'hapiDbPass';
CREATE DATABASE hapi_dstu3;
GRANT ALL PRIVILEGES ON hapi_dstu3.* to 'hapiDbUser'@'localhost';
FLUSH PRIVILEGES;
```
Success! Everything looks good thus far.

4. Update the `hapi.properties` file as indicated in the README
```
datasource.driver=com.mysql.cj.jdbc.Driver
datasource.url=jdbc:mysql://localhost:3306/hapi_dstu3
datasource.username=hapiDbUser
datasource.password=hapiDbPass
hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
```
5. Save and `mvn install`....
`[ERROR] Failed to execute goal org.apache.maven.plugins:maven-failsafe-plugin:2.21.0:verify (default) on project hapi-fhir-jpaserver-starter: There are test failures.`

6. Retrieve log from `target/classes/failsafe-reports`. Here's the critical error message in the log file that I'm seeing (happy to provide the entire file contents if helpful):
`java.sql.SQLException: Cannot create PoolableConnectionFactory (Cannot load connection class because of underlying exception: com.mysql.cj.core.exceptions.WrongArgumentException: Malformed database URL, failed to parse the main URL sections.)`

I've reviewed the database URL ad nauseam and have been unable to identify any apparent issues. I've also attempted to connect to an RDS MySQL instance to no avail. I'm able to connect to the DB instance without issue using both the CLI and a SQL client. Additionally, I had a developer colleague attempt to connect this to MySQL following the README instructions as a sanity check and encountered the same error.

--- a/README.md
+++ b/README.md
@@ -34,6 +34,8 @@ To configure the starter app to use MySQL, instead of the default Derby, update 
 * datasource.url=jdbc:mysql://localhost:3306/hapi_dstu3
 * hibernate.dialect=org.hibernate.dialect.MySQL5InnoDBDialect
 
+Because the integration tests within the project rely on the default Derby database configuration, it is important to either explicity skip the integration tests during the build process, i.e., `mvn install -DskipTests`, or delete the tests altogether. Failure to skip or delete the tests once you've configured MySQL for the datasource.driver, datasource.url, and hibernate.dialect as outlined above will result in build errors and compilation failure.
+
 It is important to use MySQL5Dialect when using MySQL version 5+.
 
 # Customizing The Web Testpage UI

